napi.h

```diff
@@ -244,6 +245,7 @@ namespace Napi {
 // released in once it is no longer experimental
 #if (NAPI_VERSION > 2147483646)
     bool IsBigInt() const;      ///< Tests if a value is a JavaScript bigint.
+    bool IsDate() const;
```

Comment:
Could you add comment like in other methods? Something like: `///< Tests if a value is a JavaScript date.`